generators/workspace/templates/plain/config/webpack.dev.js

```diff
@@ -0,0 +1,305 @@
+        flagIncludedChunks: false, // only in dev mode
+        occurrenceOrder: false, // only in dev mode
+        concatenateModules: false, // only in dev mode
+        moduleIds: 'hashed', 
+        chunkIds: 'named', // only in dev mode
+        runtimeChunk: 'single', //one runtime instance for all entries
+        splitChunks: {
+            chunks: 'all',
+            cacheGroups: {
+                workspace: {
+                    test: workspaceRegex,
+                    priority: -5,
+                    name(module) {
+                        const key = Object.keys(mergedVendors).find((key) => vendorWorkspaceRegex[key].test(module.context));
+                        return key ? key : 'workspace';
+                    }
+                },
+                vendors: {
+                    test: /[\\/]node_modules[\\/]/,
+                    priority: -10,
+                    name(module) {
+                        const key = Object.keys(mergedVendors).find((key) => vendorRegex[key].test(module.context));
+                        return key ? key : 'vendors';
+                    }
+                }
+            }
+        },
+    },
+    plugins: [
+        new CleanWebpackPlugin({
+            cleanOnceBeforeBuildPatterns: [
+                '**/*',
+                path.join(process.cwd(), '../bundles/**/*')
+            ]
+        }),
+        ...HtmlWebpackPlugins,
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify(envMode),
+            'process.env.__VERSION__': JSON.stringify(appPkg.version),
+            'process.env.__LICENSE__': JSON.stringify(appPkg.license),
+            'process.env.__BUILD_ID__': JSON.stringify(buildId),
+            'process.env.__APP_CONTEXT__': JSON.stringify('/'),
+            'process.env.__DEBUG__': JSON.stringify(isDev)
+        }),
+        new CopyWebpackPlugin({
+            patterns: [
+                {
+                    from: workspaceMetaDataFile, to: base + '/bundles/phoveaMetaData.json',
+                    //generate meta data file
+                    transform(content, path) {
+                        return webpackHelper.generateMetaDataFile({buildId}, resolve(__dirname, '../' + defaultApp));
+                    }
+                },
+                //use package-lock json as buildInfo
+                {from: workspaceBuildInfoFile, to: base + '/bundles/buildInfo.json'}
+            ]
+        }),
+        //for debugging issues
+        /*new BundleAnalyzerPlugin({
+            // set to 'server' to start analyzer during build
+            analyzerMode: 'disabled',
+            generateStatsFile: true,
+            statsOptions: {source: false}
+        })*/
+    ],
+    stats: {
+        //for source-map-loader: lineup/js warnings
+        warningsFilter: [/Failed to parse source map/],
+    }
+};
+//console.log(JSON.stringify(config, null, '\t'));
+module.exports = config;
```

Comment:
Missing new line at the end of the file.

```suggestion
module.exports = config;

```